ios/ImagePickerUtils.m

```diff
@@ -90,6 +91,14 @@ + (NSString*) getFileType:(NSData *)imageData
         return @"png";
       case firstByteGif:
         return @"gif";
+      case firstByteHeic:
+       if(isCamera)
+        {
+        return @"jpg";
+        }
+        else{
+        return @"heic";
```

Comment:
If you can fix formatting, then I can get this merged.

Reply:
@Johan-dutoit I think it is a hacky solution. you can see #1788 this for more detail